### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by blahblahbloopster**
Did some refactoring and added comments.

### CODE REVIEW

File: core/src/mindustry/world/Block.java [hunk 1]
```diff
@@ -34,9 +34,13 @@
 
 import static mindustry.Vars.*;
 
+/** Defines a type of block, does not store state. */
 public class Block extends UnlockableContent{
+    /** Weather this block can contain items. */
     public boolean hasItems;
+    /** Weather this block can contain liquids. */
```
Comment (DeltaNedas) [inline core/src/mindustry/world/Block.java:41]: Pissing rain but it's not too bad

File: core/src/mindustry/world/Tile.java [hunk 2]
```diff
@@ -446,7 +446,7 @@ public Tile getNearby(int rotation){
         return null;
     }
 
-    public Building getNearbyEntity(int rotation){
+    public Building getNearbyBuilding(int rotation){
```
Comment (DeltaNedas) [inline core/src/mindustry/world/Tile.java:449]: I think just `adjacentBuild` would be better, `getNearbyBuilding` is a mouthful and adjacent describes it better.

**Review by DeltaNedas — COMMENTED** [review]
j

**Comment by sk7725** [comment]
Whether, not weather

one gives you a choice, and one punishes you for choosing wrong on whether to bring an umbrella with you today.

**Comment by Anuken** [comment]
Some of these have grammatical/spelling issues, and some of them are just wrong. I'd rather do it myself.

**Pull request closed without merging**